src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/VisitNotificationEventRepository.kt

```diff
@@ -20,4 +20,21 @@ interface VisitNotificationEventRepository : JpaRepository<VisitNotificationEven
     bookingReference: String,
     notificationEvent: NotificationEventType,
   ): Boolean
+
+  @Query(
+    "SELECT vne.* FROM visit_notification_event vne " +
+      " JOIN visit v on v.reference  = vne.booking_reference  " +
+      " JOIN prison p on p.id  = v.prison_id " +
+      " WHERE  v.visit_start >= NOW() " +
```

Comment:
a suggestion, can be ignored - we could pass the date as a parameter.

Reply:
I thought about this, there is no requirement for this, you always pick up on this :-)